tree/20_cfe_basics/services.cf

```diff
@@ -209,13 +196,13 @@ bundle agent ncf_services(service, action)
                                 strcmp("is-active-process", "${action}"),
                                 strcmp("is-enabled", "${action}")
                               };
-      "is_start_action"         expression => regcmp(".*start.*", "${action}");
+      "is_start_action"         expression => regcmp(".*start$", "${action}");
```

Comment:
it matches restart also. Shouldn't we have a case for that ?

Reply:
restart also requires clearing the failt status, but I noticed that it also matches "is-started" which does not.